source/reference/verification/custom-manual.txt

```diff
@@ -0,0 +1,86 @@
+.. _c2c-custom-manual-verification:
+
+==================================
+Custom or Manual Verification Page
+==================================
+
+.. default-domain:: mongodb
+
+.. facet::
+   :name: genre
+   :values: tutorial
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+.. _c2c-document-comparison: 
+
+Document Comparison
+-------------------
```

Comment:
```suggestion
Comparing Documents
-------------------
```
I'm not a fan of nominalization in titles, since it sounds robotic. I don't know for sure if this is anywhere in our style guide, so feel free to get a second opinion on these heading suggestions.